src/meta/test/ProcessorTest.cpp

```diff
@@ -1947,7 +2042,7 @@ TEST(ProcessorTest, AlterEdgeTest) {
     cpp2::Schema changeSch;
     cpp2::ColumnDef column;
     column.name = "edge_0_col_10";
-    column.type.type_ref() = PropertyType::DOUBLE;
+    column.type.type_ref() = PropertyType::INT64;
```

Comment:
Note that the current column type of `col_10` is int64. The failure should come from changing col on ttl. So I keep the column type the same.